tests/test_expectation_maximization.py

```diff
@@ -254,12 +254,23 @@ def test_insert_slice(test_ir, n_pix):
     )
 
     slices, xyz_rotated_planes = test_ir.generate_slices(
-        map_plane_ones, xy_plane, n_pix, rot_90deg_about_y
+        map_plane_ones, xy_plane, rot_90deg_about_y
```

Comment:
This test seems general for any rotation. I would run it on several random rotations instead of a hard coded `rot_90deg_about_y`

Reply:
You could do a hard coded check for an inserted slice. e.g. rotate the vertical plane down the a horizontal, and then construct the map it is supposed to match.

Reply:
This is the hardcoded test, it just skips some tedium by generating slices with the function - it doesn't generalize to other rotations because the final matrix map_plane_ones needs to be easily predictable.

I'll think about other test options